doc/building.md

```diff
@@ -52,76 +52,17 @@ Dependency list:
-
-
-### Prerequisite steps for Mac OS X users (OS X 10.10 Yosemite)
-
- - Install [Homebrew](http://brew.sh). Other package managers and building from source might also work, but aren't supported as of now. If you figure out fixes for those other options, you're welcome to open pullrequests, as long as the Homebrew build doesn't break.
- - `brew update`
- - `brew install caskroom/cask/brew-cask`
- - `brew tap caskroom/fonts`
- - `brew tap homebrew/versions`
- - `brew tap homebrew/python`
- - `brew update` (yes, again)
- - `brew cask install font-dejavu-sans`
- - `brew install python3 libepoxy freetype fontconfig harfbuzz cmake sdl2 sdl2_image opus opus-tools opusfile`
- - `brew install numpy --with-python3`
- - `brew install pillow --with-python3`
- - `brew install qt5`
- - Choose and Install a compatible compiler:
-  - For clang 3.5: `brew install llvm35`
-  - For gcc 4.9: `brew install gcc49`
- - Permanently add `/usr/local/bin` to your `$PATH` environment variable
- - Permanently add `/usr/local/lib/pkgconfig` and `/usr/local/lib` to your `$PKG_CONFIG_PATH` environment variable
- - Then run `pip3.5 install pygments cython`
-
-
-### Prerequisite steps for Arch Linux users
-
-> NOTE: [aur](https://aur.archlinux.org/packages/openage-git/) exists, to install via **yaourt** run `yaourt -S openage-git`.
-
-This command should provide required packages for Arch Linux installation:
-
-`sudo pacman -S --needed python python-pillow python-numpy python-pygments cython libepoxy ttf-dejavu freetype2 fontconfig harfbuzz cmake sdl2 sdl2_image opusfile opus-tools python-pylint qt5-declarative qt5-quickcontrols`
-
-If you don't have a compiler installed, you can select between these commands to install it:
- - `sudo pacman -S --needed gcc`
- - `sudo pacman -S --needed clang`
-
-You can install both compilers and select the one to be used by `./configure`.
+There are some prerequisite steps that need to be performed so *openage* can be succesfully build. Those steps vary from platform to platform, and are described bellow for some of the most common ones:
 
+- [Prerequisite steps for Ubuntu users (Ubuntu 15.04)](build_instructions/ubuntu_15.04.md);
+- [Prerequisite steps for Fedora users (Fedora 20, 21)](build_instructions/fedora_20_21.md);
+- [Prerequisite steps for Fedora users (Fedora 22)](build_instructions/fedora_22.md);
+- [Prerequisite steps for openSUSE users (openSUSE 13.2)](build_instructions/opensuse_13.2.md);
+- [Prerequisite steps for openSUSE users (openSUSE Tumbleweed)](build_instructions/opensuse_tumbleweed.md);
+- [Prerequisite steps for Mac OS X users (OS X 10.10 Yosemite)](build_instructions/os_x_10.10_yosemite.md);
+- [Prerequisite steps for Arch Linux users](build_instructions/arch_linux.md).
```

Comment:
i think it's best to just write it like

```
- [Ubuntu (>=15.04)](...)
```

because the "users" and "prerequisite steps" is very redundant, the `;` or `.` is also not needed i'd say.